packages/shopify-codemod/transforms/remove-redundant-returns-from-addeventlistener.js

```diff
@@ -0,0 +1,28 @@
+export default function removeAddEventListener(file, api) {
+  const j = api.jscodeshift;
+  return j(file.source)
+    .find(j.CallExpression, {
+      callee: {
+        type: 'MemberExpression',
+        property: {
+          name: 'addEventListener',
+        },
+      },
+    })
+    .forEach((nodes) => {
+      const functionExpression = nodes.value.arguments[1];
+      const {body} = functionExpression.body;
+      const lastStatement = body[body.length - 1];
+      if (lastStatement.type === 'ReturnStatement') {
+        if (lastStatement.argument === null) {
+          body.pop();
```

Comment:
Probably want to return here to avoid the undefined expression issue Gord mentioned.